import-service/serverless.yml

```diff
@@ -0,0 +1,50 @@
+service: import-service
+
+provider:
+  name: aws
+  runtime: nodejs12.x
+  stage: dev
+  region: eu-west-1
+  environment:
+    BUCKET: recycle-buckets-shop-files
+    REGION: ${self:provider.region}
+  iamRoleStatements:
+    - Effect: "Allow"
+      Action:
+        - "s3:ListBucket"
+      Resource: "arn:aws:s3:::recycle-buckets-shop-files"
+    - Effect: "Allow"
+      Action:
+        - "s3:*"
+      Resource: "arn:aws:s3:::recycle-buckets-shop-files/*"
+
+plugins:
+  - serverless-webpack
+
+custom:
+  webpack:
+    webpackConfig: 'webpack.config.js'
+    includeModules: true
```

Comment:
btw, you can use

```
forceExclude:
            - aws-sdk
```

to reduce bundle size

Reply:
As I understand webpack plugin will include only deps from `dependencies` section and not from `devDependencies`. Isn't it?

Reply:
idd, have to take a rest :DDD you already put it into devDependencies
sry